src/RepositoryApiClient.cs

```diff
@@ -11,7 +11,9 @@ namespace Laserfiche.Repository.Api.Client
     /// </summary>
     public class RepositoryApiClient : IRepositoryApiClient
     {
-        private const string _defaultBaseAddress = "https://dummy.example.com/repository/";
```

Comment:
I was testing using the `_defaultBaseAddress` and it seems to work with the LfdsUsernamePasswordHandler. Is having different dummy base address necessary?

Reply:
You are right. It also works.